### ISSUE TITLE
use this project for Drupal side

### ISSUE THREAD

**Issue by ccprog** [refugeehackathon/refugeetransit-website#1]
I'd say we follow this proposal for using the project: https://github.com/refugeehackathon/refugeetransit-design/issues/1 and put the public project description here on [GitHub Pages](https://pages.github.com/)

**Comment by ligi** [comment]
IMHO we should only have one website ( the drupal one ) and here is the right place for it

**Comment by ccprog** [comment]
I think it was @nebulak opposing yesterday. He wanted "a simple, static page". I'm fine with both.

**Comment by mischazip** [comment]
@ligi : The Drupal page is used as our backend(+API-Endpoint) and gets filled with a lot of articles in different languages... So at the moment, if you visit the normal Drupal-Website it's just a big chaos.

I would create the info page/website using Github pages to seperate our backend/api and the normal website. IMHO, if we only use Drupal I see the risk that someone who just wants to edit a page, could make a mistake and makes our API temporarily unusable...

**Comment by t2d** [comment]
I agree, that we should have a very simple landing page. Probably, this won't be drupal.

> IMHO, if we only use Drupal I see the risk that someone who just wants to edit a page, could make a mistake and makes our API temporarily unusable...

I think, this is a separate problem of the API. It should fail softly. Opening own issue: https://github.com/refugeehackathon/refugeetransit-api/issues/4
